test/e2e-cypress/specs/apple-pay.cy.ts

```diff
@@ -0,0 +1,10 @@
+// TODO: configure to only run on Safari, add more test cases
```

Comment:
I think for now we can forgo the apple pay test, due to the safari only requirement.

Reply:
Agreed! Removed.